runtime/oti/VMHelpers.hpp

```diff
@@ -2154,13 +2177,13 @@ class VM_VMHelpers
 #endif /* JAVA_SPEC_VERSION >= 19 */
 
 	static VMINLINE UDATA
-	walkContinuationStackFramesWrapper(J9VMThread *vmThread, j9object_t continuationObject, J9StackWalkState *walkState, bool isConcurrentGC, bool isGlobalGC)
+	walkContinuationStackFramesWrapper(J9VMThread *vmThread, j9object_t continuationObject, J9StackWalkState *walkState, bool isConcurrentGC, bool isGlobalGC, bool beingMounted)
 	{
 		UDATA rc = J9_STACKWALK_RC_NONE;
 #if JAVA_SPEC_VERSION >= 19
 		J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, continuationObject);
 		if (isConcurrentGC && (NULL != continuation)) {
```

Comment:
now that preMount is after sync (more importantly after struct create), think if it's valid to make this non-null condition an assert

Reply:
if so, check the same for the check before we call exitConcurrentGCScan